pkg/internal/cluster/providers/provider/common/proxy.go

```diff
@@ -48,7 +52,11 @@ func GetProxyEnvs(cfg *config.Cluster) map[string]string {
 	}
 	// Specifically add the cluster subnets to NO_PROXY if we are using a proxy
 	if len(envs) > 0 {
-		noProxy := strings.Join([]string{envs[NOProxy], cfg.Networking.ServiceSubnet, cfg.Networking.PodSubnet}, ",")
```

Comment:
previously we would do something like `,10.0.0.0/24,12.0.0.0/24` if `NO_PROXY` was not found which technically seems to have worked (?) but at best seems strange and at worse seems likely to break some application, given how under-defined this environment variable "API" is.

Reply:
I remember vaguely to have verified it ... but don't remember how 😅